corehq/apps/app_manager/templates/app_manager/v1/app_view.html

```diff
@@ -188,6 +188,48 @@
             }));
         });
     </script>
+    <script>
+        $("#copy form button").click(function() {
+            var $submit = $(this),
+                $form = $submit.closest("form"),
+                domain = $form.find("#id_domain").val(),
+                $modal = $("#copy-feature-flags");
+            {% if request|toggle_enabled:"SUPPORT" %}
```

Comment:
Shouldn't this be enabled for all dimagi users?  You could use `user.is_superuser`, since those are the users who can access feature flags.